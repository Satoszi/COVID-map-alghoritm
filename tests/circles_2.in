10 10 2 32
circles_2
+--------+
|+------+|
||+----+||
|||+--+|||
||||++||||
||||++||||
|||+--+|||
||+----+||
|+------+|
+--------+
