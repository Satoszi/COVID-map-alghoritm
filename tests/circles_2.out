9 9
6 9
3 9
0 9
6 8
3 8
3 7
1 7
9 6
8 6
7 6
6 6
3 6
0 6
2 5
4 4
1 4
9 3
8 3
7 3
6 3
3 3
0 3
5 2
2 2
7 1
4 1
1 1
9 0
6 0
3 0
0 0
